### ISSUE TITLE
Good sources to start learning JavaScript and some tips for beginners.

### ISSUE THREAD

**Issue by JiteshDhalwani** [anuraghazra/anuraghazra#5]

**Comment by HarshKapadia2** [comment]
Yeah would definitely love it if you could share some resources or some ways to get better at vanilla JS.

**Comment by iRupam** [comment]
I agree with you guys. BTW I am learning basic JS from [https://www.freecodecamp.org/learn/](url).

**Comment by Aaditay** [comment]
@JiteshDhalwani  @HarshKapadia2  @iRupam  

This is a good resource to learn JS : https://javascript.info
Happy Learning :)

**Comment by anuraghazra** [comment]
I have some resources which you can get started but i did not shared it because i think it just takes 1 google search to found very useful resources but anyways here are some :-

*(note: this resources i collected long ago just to copy paste to people who asked me the same question and i did not updated anything till then, so feel free to add anything)*

## Youtube Channels:
- TheCodingTrain
- TheNetNinja
- Academind
- BradTravery
- Codecademy
- freeCodecamp
- learn.CodeAcademy
- leveluptuts
- CodeWithKyle
- DevTips

## Articles & Sites:
- https://www.javascriptstuff.com/
- http://javascript.info/
- https://developer.mozilla.org/en-US/docs/Web/JavaScript
- https://www.codecademy.com/learn/introduction-to-javascript
- https://www.geeksforgeeks.org/javascript-tutorial/
- https://medium.freecodecamp.org/the-complete-javascript-handbook-f26b2c71719c
- https://medium.freecodecamp.org/learn-these-javascript-fundamentals-and-become-a-better-developer-2a031a0dc9cf
- https://github.com/getify/You-Dont-Know-JS
- https://github.com/ryanmcdermott/clean-code-javascript
- https://github.com/micromata/awesome-javascript-learning
- https://vegibit.com/javascript-learning-tutorials/

i hope these are enough for you to get started with javascript. 
javascript is amazing and there are lot of learning resources around internet but i pick this for you. 

And i would recommend you one more thing Mozilla Documentations MDN is amazing resource for everything about web development and javascript follow their website and if you get stuck firstly look for stackoverflow and then go to MDN

and once more essential skill you have to master in is Reading Articles and Reading Documentation, if you can read docs you can learn anything.

Firstly learn the basics of javascript then go to ES6 more advanced concepts of javascript then dive deeper into FrontEnd frameworks React,Angular,Vue etc etc. and if you want you can also go for Nodejs.

And the best way to learn javascript is do lots and lots of experimentation..... make projects and experiment with codes

Thanks, i hope this helps.

**Comment by iRupam** [comment]
@anuraghazra I think DevTips channel is no longer active. 😥

**Comment by anuraghazra** [comment]
> @anuraghazra I think DevTips channel is no longer active.

yeah as i said i did not updated the list :(

**Comment by iRupam** [comment]
@anuraghazra I have one question. Should we use [?](jsfiddle.net) Also, do you use it personally?

**Comment by iRupam** [comment]
Should we use [jsfiddle.net](jsfiddle.net)? Do you use it personally?

**Comment by anuraghazra** [comment]
No i don't i prefer codesandbox

**Comment by JiteshDhalwani** [comment]
Any thoughts on FCC or The Odin Project's web development courses?

**Comment by anuraghazra** [comment]
I don't prefer courses personally, but it depends on person to person. :)
